consumer_test.go

```diff
@@ -1424,8 +1424,8 @@ func TestConsumer_isWithinCapcity(t *testing.T) {
 				opts.Dir = ""
 				return
 			},
-			wantOk:  false,
-			wantErr: true,
+			wantOk:  true,
+			wantErr: false,
```

Comment:
wouldnt you want an error for error walking?

Reply:
Fantastic catch and completely valid concern, I should have made these changes during this PR:
https://github.com/mojura/kiroku/pull/28/files

Then it would have made a lot more sense